ocaml/idl/datamodel.ml

```diff
@@ -1216,6 +1216,8 @@ let _ =
     ~doc:"The update precheck stage failed: the server is of an incorrect version." ();
   error Api_errors.update_precheck_failed_out_of_space ["update"; "available_space"; "required_space "]
     ~doc:"The update precheck stage failed: the server does not have enough space." ();
+  error Api_errors.update_precheck_failed_gpgkey_not_imported ["update"]
+    ~doc:"The update precheck stage failed: RPM package validation requires a GPG key that is not imported on the host." ();
```

Comment:
I would say any of those would be reasonable.

"...*is* not imported..." is like saying some wood *is* not varnished or a machine *is* not switched on or a light-bulb *is* not installed in a fitting.

In principle there could be a key which *has been* imported but then removed.

Talking about the key being present avoids the question over tense, but it a bit less specific about what kind of presence is meant, and is a bit further from the language used in the original error message from the gpg tool.